### PR TITLE
EarlyAccess document

### PR DESCRIPTION
**Pull request by btardif**

### CODE REVIEW

**Comment by btardif** [comment]
FYI @bradygaster

File: Runtime_Support/dot_net_core.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ Once a version of .NET Core has reached it's end of life (EOL) it will no longer
 
 Existing applications configured to target a runtime version that has reached EOL should not be affected.
 
+## .NET 5 Early Access limitations
+
+During the early access period, .NET 5 apps will be subject to the **Early Access** limitations listed in the [early access support document](./early_access.md).
```
Comment (bradygaster) [inline Runtime_Support/dot_net_core.md:15]: added comments in the early access md, too.

File: Runtime_Support/dot_net_core.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ Once a version of .NET Core has reached it's end of life (EOL) it will no longer
...
+
+[Application Insights](https://azure.microsoft.com/services/monitor) is currently not supported for .NET 5 apps, we are working with the Application Insights team to resolve this.
```
Comment (bradygaster) [inline Runtime_Support/dot_net_core.md:17]: cc @shirhatti as an FYI that @btardif's article on early access is going to call out the lack of support for App Insights. I wanted you to review this. I'll be linking to Byron's doc here and the one on general early access info, so I don't know how deep i'll go in the actual .NET blog post. If you think I should summarize the whole .NET piece of this in our blog post **instead** of linking people over here, I'm okay with that too, just let me know your opinion.

File: Runtime_Support/dot_net_core.md [hunk 3]
```diff
@@ -1,25 +1,34 @@
-# .NET Core on App Service
+# .NET on App Service
```
Comment (bradygaster) [inline Runtime_Support/dot_net_core.md:1]: 👍

File: Runtime_Support/dot_net_core.md [hunk 4]
```diff
@@ -1,25 +1,34 @@
...
 
-## .NET Core Update Policy
+## .NET Update Policy
 
-App Service upgrades the underlying .NET Core runtime and SDK of your application as part of the regular platform updates. As a result of this update process, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
+App Service upgrades the underlying .NET runtime and SDK of your application as part of the regular platform updates. As a result of this update process, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
```
Comment (bradygaster) [inline Runtime_Support/dot_net_core.md:5]: "as a result of this update" - you already said "process"

File: Runtime_Support/early_access.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# Early Access Runtime
+
+**Early Access** is a new feature of App Service that allows for the rapid delivery of updates and new versions of the programing languages/runtime and SDK supported by the platform. As a developer you will get access to the latest new releases, without having to wait for an App Service release cycle to include support.
```
Comment (bradygaster) [inline Runtime_Support/early_access.md:3]: **Early Access** is a new feature of Azure App Service that allows for the rapid delivery of updates and new versions of various-language programming runtimes and SDKs supported by the platform. As a developer, you will get access to the latest new releases without having to wait for an App Service release cycle to include support.

**Review by bradygaster — APPROVED** [review]
awesome work! SUPER exciting.

File: Runtime_Support/dot_net_core.md (hunk 4 above)
Comment (btardif) [reply Runtime_Support/dot_net_core.md:5]: fixed

File: Runtime_Support/dot_net_core.md (hunk 1 above)
Comment (btardif) [reply Runtime_Support/dot_net_core.md:15]: 👍🏽

File: AzWebAppUP/runtime_detection.md [hunk 6]
```diff
@@ -29,6 +29,23 @@ Node detection looks for the existence of the `package.json` or `sever.js` or `i
 
 ASP .NET and .NET Core detection is done by looking for the existence of `*.csproj` in the root folder of the project.
 
+### .NET 5
+
+Detection logic for .NET 5 apps will look into your `*.csproj` file in the current folder.
+
+Detection will then check value set for  `TargetFramework` element looking for `net5.0` string.
```
Comment (btardif) [inline AzWebAppUP/runtime_detection.md:36]: FYI @panchagnula and @calvinsID 

Keep me honest here

File: Runtime_Support/dot_net_core.md [hunk 7]
```diff
@@ -1,25 +1,35 @@
...
 Existing applications configured to target a runtime version that has reached EOL should not be affected.
 
+## .NET 5 Early Access limitations
+
+During the early access period, .NET 5 apps will be subject to the **Early Access** limitations listed in the [early access support document](./early_access.md).
+
+[Application Insights](https://azure.microsoft.com/services/monitor) is currently not supported for .NET 5 apps, we are working with the Application Insights team to resolve this.
+
+Visual Studio and Visual Studio for Mac will not support creating new .NET 5 Apps on App Service through the IDE at launch. As a workaround .NET 5 Apps can be created from the Azure Portal or Azure CLI.
+Publishing content to a .NET 5 works as expected. There is schedule update  that will remove this limitation.
```
Comment (btardif) [inline Runtime_Support/dot_net_core.md:20]: FYI @bradygaster let me know if you want to change anything here

Comment (bradygaster) [reply Runtime_Support/dot_net_core.md]: you have an extra space between update and that

Comment (btardif) [reply Runtime_Support/dot_net_core.md]: fixed

File: Runtime_Support/dot_net_core.md [hunk 8]
```diff
@@ -17,7 +17,7 @@ During the early access period, .NET 5 apps will be subject to the **Early Acces
 [Application Insights](https://azure.microsoft.com/services/monitor) is currently not supported for .NET 5 apps, we are working with the Application Insights team to resolve this.
 
 Visual Studio and Visual Studio for Mac will not support creating new .NET 5 Apps on App Service through the IDE at launch. As a workaround .NET 5 Apps can be created from the Azure Portal or Azure CLI.
-Publishing content to a .NET 5 works as expected. There is schedule update  that will remove this limitation.
+Publishing content to a .NET 5 app works as expected. There is a schedule update  that will remove this limitation.
```
Comment (bradygaster) [inline Runtime_Support/dot_net_core.md]: 👍

**Review by bradygaster — APPROVED** [review]

File: AzWebAppUP/runtime_detection.md (hunk 6 above)
Comment (calvinsID) [reply AzWebAppUP/runtime_detection.md:36]: Yup looks good

**Comment by btardif** [comment]
Pushing this live

**Pull request merged**